datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyMulticastStrategy.java

```diff
@@ -301,11 +302,16 @@ public void testSameTaskIsNotAssignedToMoreThanOneInstance() {
     Assert.assertEquals(newAssignmentTasks.size(), instances.length * numDatastreams);
   }
 
+  @NotNull
+  private StickyMulticastStrategy createStickyMulticastStrategyObject(Optional<Integer> maxTasks) {
```

Comment:
not sure I understand why a function was necessary for this. Can't you directly call "new StickyMulticastStrategy(<maxtasks>)"? 
I also see that in this file you haven't called this function consistently. Some places you call "new StickyMulticastStrategy" directly, some places use this function. Pick one thing and use it consistently.